File: src/game/tools/rpc/output/wrap/DailyGiftUpdateDataWrap.cs
using System;
using Common.Net.Simple;

namespace Game 
{
	public class DailyGiftUpdateDataWrap : IWrapper
	{	
		public int GiftIdx; // 当前领取的idx
		public int Status; // 更新状态
		
		public void Decode(Packet pck)
		{	
			GiftIdx = pck.GetInt(); 
			Status = pck.GetInt(); 
		}
		
        public void Encode(Packet pck)
        {	
        	pck.PutInt(GiftIdx); 
        	pck.PutInt(Status); 
        }
	}
}

File: src/game/tools/rpc/output/wrap/DailyGiftUpdateDataWrap.go
package wraps

import (
	rpc "tipcat.com/common/rpc/simple"
)

type DailyGiftUpdateDataWrap struct {
	
	GiftIdx int32 // 当前领取的idx
	Status int32 // 更新状态
	rpc.Wrapper
}

func (w *DailyGiftUpdateDataWrap)Decode(pck *rpc.Packet) rpc.Wrapper {
	
	w.GiftIdx = pck.PopInt32(); 
	w.Status = pck.PopInt32(); 
	return w
}

func (w *DailyGiftUpdateDataWrap)Encode(pck *rpc.Packet) {
	
	pck.PutInt32(w.GiftIdx) 
	pck.PutInt32(w.Status) 
}

